ui/lib/citizen/useCitizenEmail.tsx

```diff
@@ -20,23 +21,34 @@ export default function useCitizenEmail(nft: any) {
         'formId'
       ).value
 
-      const res = await fetch(
-        `/api/typeform/response?formId=${process.env.NEXT_PUBLIC_TYPEFORM_CITIZEN_FORM_ID}&responseId=${formResponseId}`,
-        {
-          method: 'POST',
-          headers: {
-            Authorization: `Bearer ${accessToken}`,
-          },
-        }
-      )
-
-      const data = await res.json()
-
-      const citizenEmail = data?.answers?.find(
-        (a: any) => a.field.id === 'LzGGOX3e8Sfv'
-      )?.email
-
-      setEmail(citizenEmail)
+      try {
+        const citizenFormV1Email = await fetchEmail(
+          process.env.NEXT_PUBLIC_TYPEFORM_CITIZEN_FORM_ID as string,
+          formResponseId,
+          'LzGGOX3e8Sfv',
+          accessToken
+        )
+
+        const citizenShortFormEmail = await fetchEmail(
+          process.env.NEXT_PUBLIC_TYPEFORM_CITIZEN_SHORT_FORM_ID as string,
+          formResponseId,
+          'JEiG9XCW6M73',
+          accessToken
+        )
+
+        const citizenEmailFormEmail = await fetchEmail(
+          process.env.NEXT_PUBLIC_TYPEFORM_CITIZEN_EMAIL_FORM_ID as string,
+          formResponseId,
+          'Z3IMkpvJUfdl',
```

Comment:
do you want to move this to const file?

Reply:
also a few others in useTeamEmail

Reply:
> do you want to move this to const file?

they should never change and won't be used anywhere else